samples/websocket/echoServer/bundle/resources/carbon.product

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why do we need a carbon.product file here? Bundle is intended to be copied by the user to the dropins folder. This should be specified in the readme